src/roles/common/tasks/wait-for-ssh.yml

```diff
@@ -20,3 +20,8 @@
   delay: "{{ delay | default(10) }}"
   remote_user: "{{ username_on_the_host }}"
   become: false
+
+- name: Check output of SSH test
+  assert:
```

Comment:
To get to this assert, we need to add ignore_errors: True in the above SSH command. Else this assert will never be printed.